.github/CODEOWNERS

```diff
@@ -126,7 +126,7 @@
 #/avm/res/resources/tags/ @Azure/avm-res-resources-tags-module-owners-bicep @Azure/avm-core-team-technical-bicep
 /avm/res/search/search-service/ @Azure/avm-res-search-searchservice-module-owners-bicep @Azure/avm-core-team-technical-bicep
 /avm/res/service-bus/namespace/ @Azure/avm-res-servicebus-namespace-module-owners-bicep @Azure/avm-core-team-technical-bicep
-#/avm/res/service-fabric/cluster/ @Azure/avm-res-servicefabric-cluster-module-owners-bicep @Azure/avm-core-team-technical-bicep
+/avm/res/service-fabric/cluster/ @Azure/avm-res-servicefabric-cluster-module-owners-bicep @Azure/avm-core-team-technical-bicep
```

Comment:
Soft-block for this file until the GitHub teams are confirmed in the hierachy

Reply:
Hey @snoejovich, from where I'm standing the PR looks ready to be merged, but the team is either not yet created, or not yet added in the correct location in the hierachy so that the CODEOWNERS file will work as expected (hence it shows errors in the `Files changed` view of the Pull Request. You can find more information on what exactly to do in [this](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-snfr20---category-contributionsupport---github-teams-only) spec.

Reply:
The parent team assignment was missing. I've added the avm-technical-reviewers-bicep parent team and am awaiting  the approval.